src/state/api.js

```diff
@@ -18,9 +19,20 @@ export async function getPingData(appName, pingName) {
 }
 
 export async function getMetricData(appName, metricName) {
+  let updatedMetricName = metricName;
+
+  Object.keys(UBLOCK_ORIGIN_PRIVACY_FILTER).forEach((filter) => {
+    if (updatedMetricName.includes(filter)) {
+      updatedMetricName = updatedMetricName.replace(
```

Comment:
One note since I looked it up: JS `replace` will only replace the first occurrence, whereas the Python `replace` used in `etl/glean_etl.py` will replace all occurrences by default. I don't think it's worth worrying about (seems pretty unlikely that there will be a metric with the same blocked string twice), but just so you're aware on the off chance it does come up in the future. Possible solutions would be to use regex (with /g) or nested loops to ensure you get all the instances of the filter key.